etc/ci/upload_nightly.sh

```diff
@@ -12,7 +12,12 @@ usage() {
 
 
 upload() {
-    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    local nightly_upload_dir package_filename package_upload_path
```

Comment:
Actually, all three of these variables can use the `local -r` form as below, and we don't need this line.